src/app/tabs/test-tab.js

```diff
@@ -495,7 +523,8 @@ module.exports = class TestTab extends ViewPlugin {
       <div class="${css.testTabView} px-2" id="testView">
         <div class="${css.infoBox}">
           <p class="text-lg"> Test your smart contract in Solidity.</p>
-          <p> Click on "Generate" to generate a sample test file.</p>
+          <p> Click on "Generate" to generate a sample test file in.</p>
```

Comment:
This sentence should end with a : not a .
Or maybe this will be more clear:
Click on "Generate" to generate a sample test file in this folder:
Or put a label and use this sentence:
Choose a folder & click "Generate" for a sample test file.
<img width="315" alt="image" src="https://user-images.githubusercontent.com/6050001/85428394-210e9580-b54b-11ea-9b5d-60f3fb942a20.png">

Reply:
thanks :)